daml-foundations/daml-ghc/test-src/DA/Test/ShakeIdeClient.hs

```diff
@@ -739,3 +740,12 @@ stressTests mbScenarioService = Tasty.testGroup "Stress tests"
 -- | Suppress unused binding warning in case we run out of tests for open issues.
 _suppressUnusedWarning :: ()
 _suppressUnusedWarning = testCaseFails `seq` ()
+
+platformSpecificMsg :: T.Text -> T.Text
+platformSpecificMsg msg = case os of
```

Comment:
Better to use System.Info.Extra.isWindows

Reply:
Thanks, modified.